modules/policy-schema.js

```diff
@@ -0,0 +1,15 @@
+
+const utils = require("./graphman-utils");
+const Ajv2020 = require("ajv/dist/2020");
+
+module.exports = {
+    build: function () {
+        const ajv = new Ajv2020()
+        const policySchemaFile = utils.policySchemaFile();
+        const policySchema = utils.readFile(policySchemaFile);
+        const validate = ajv.compile(policySchema)
+        return validate;
```

Comment:
we should some how make loading the ajv library optionally. So that, graphman-client can continue to work other than validating the policy code.

Reply:
done